javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/index.html

```diff
@@ -0,0 +1,207 @@
+                    fill="#F5F6F7"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="playlist">
+        <div class="playlist__bar">
+          <div>
+            <svg
+              width="130"
+              height="10"
+              viewBox="0 0 130 10"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M130 1H0M130 9H0" stroke="#F1BE32" stroke-width="2" />
+            </svg>
+          </div>
+          <h2 class="playlist-title" id="playlist">Playlist</h2>
+          <div>
+            <svg
+              width="130"
+              height="10"
+              viewBox="0 0 130 10"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M130 1H0M130 9H0" stroke="#F1BE32" stroke-width="2" />
+            </svg>
+          </div>
+          <div>
+            <button
+              class="playlist__close"
+              aria-label="Playlist"
+              aria-expanded="true"
+            >
+              X
+            </button>
+          </div>
+        </div>
+        <ul id="playlist__songs" aria-labelledby="playlist"></ul>
+      </div>
+    </div>
+    <script src="scripts.js"></script>
```

Comment:
This should be `script.js` so it matches what is in the curriculum editor.
We will also have to change the actual file name too

```suggestion
    <script src="script.js"></script>
```